apps/SageAccountWeb/src/App.scss

```diff
@@ -1,6 +1,6 @@
 @use './style/variables' as Portal;
 
-@use 'synapse-react-client/src/style/main.scss' with (
+@use 'synapse-react-client/style/main.scss' with (
```

Comment:
I think Vite now uses a Sass API to resolve files the same way Vite resolves other files, so we can remove the `src/`